src/Symfony/Bundle/FrameworkBundle/Test/LoggerAssertionsTrait.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Test;
+
+use Monolog\Handler\TestHandler;
+use Monolog\Logger;
+use Monolog\LogRecord;
+
+trait LoggerAssertionsTrait
+{
+    public static function assertLogExists(string $expectedLog, string $level = Logger::DEBUG): void
+    {
+        /** @var TestHandler $logger */
+        $logger = self::getContainer()->get('monolog.handler.test');
```

Comment:
Who's in charge of declaring this test handler? The developer? 
There is no such service declared out-of-the-box AFAIK.

Should there be an option to register one, or a section added into the monolog recipe about this?

Reply:
Good catch, I've to add it in the app where I made those assertions first.

I can check if it's registered, and if not, display an info message to suggest the developer to register the test handler.